benchmarks/utils/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_library(
+    benchmark_output_utils STATIC
+    output_utils.cpp)
+target_include_directories(benchmark_output_utils PUBLIC .)
```

Comment:
Can it be `PRIVATE` ?

Reply:
Not sure that understand `.` in that case, could you specify which libraries are used

Reply:
It provides access to output_utils.hpp